Wikipedia/Code/NSUserDefaults+WMFExtensions.swift

```diff
@@ -285,12 +285,12 @@ public extension NSUserDefaults {
     }
 
     public func wmf_setDidMigrateToNewFeed(didMigrate: Bool) {
-        self.setBool(didMigrate, forKey: WMFDidMigrateToDatabaseFeedKey)
+        self.setBool(didMigrate, forKey: WMFDidMigrateToCoreDataFeedKey)
         self.synchronize()
     }
     
     public func wmf_didMigrateToNewFeed() -> Bool {
-        return self.boolForKey(WMFDidMigrateToDatabaseFeedKey)
+        return self.boolForKey(WMFDidMigrateToCoreDataFeedKey)
```

Comment:
Just to check, your migration code handles:
1. Users who have migrated the feed to Yap
2. Users who have not migrated the feed to Yap, but have migrated to the shared container
3. Users who have not migrated to the shared container
4. Users who have not migrated history to Yap